src/app/pages/orders/components/orders/orders.component.ts

```diff
@@ -32,12 +30,9 @@ export class OrdersComponent implements OnInit {
   async ngOnInit(): Promise<void> {
     await this.productsFacadeService.initOrdersPage();
 
-    this.ordersFacadeService.getOrderProducts()
-      .pipe(takeUntilDestroyed(this.destroyRef))
-      .subscribe((orders: IOrder[]) => {
-        this.orders = orders;
-        this.cdr.detectChanges();
-      });
+    this.orders = this.ordersFacadeService.getOrderProducts();
+
+    this.cdr.detectChanges();
```

Comment:
we don't need cdr if we use async pipe, async pipe automatically makes the detect changes